crates/sui-storage/src/indexes.rs

```diff
@@ -31,18 +31,17 @@ pub struct IndexStore {
 
     /// Index from object id to transactions that modified/created that object id.
     transactions_by_mutated_object_id: DBMap<(ObjectID, TxSequenceNumber), TransactionDigest>,
+
+    /// This is a map between the transaction digest and its timestamp (UTC timestamp in
+    /// **milliseconds** since epoch 1/1/1970). A transaction digest is subjectively time stamped
+    /// on a node according to the local machine time, so it varies across nodes.
+    /// The timestamping happens when the node sees a txn certificate for the first time.
+    timestamps: DBMap<TransactionDigest, u64>,
 }
 
 impl IndexStore {
     pub fn open<P: AsRef<Path>>(path: P, db_options: Option<Options>) -> Self {
-        let mut options = db_options.unwrap_or_default();
-
-        // The table cache is locked for updates and this determines the number
-        // of shareds, ie 2^10. Increase in case of lock contentions.
-        let row_cache = rocksdb::Cache::new_lru_cache(1_000_000).expect("Cache is ok");
-        options.set_row_cache(&row_cache);
-        options.set_table_cache_num_shard_bits(10);
-        options.set_compression_type(rocksdb::DBCompressionType::None);
+        let (options, point_lookup) = default_db_options(db_options, Some(1_000_000));
```

Comment:
thanks for removing this duplication!! and preserving the original behavior but I think this number just got copied from somewhere else and the default value would also be fine here. No need to change it again though.